python-checks/src/test/resources/checks/torchLoadLeadsToUntrustedCodeExecution.py

```diff
@@ -0,0 +1,20 @@
+import torch
+from torch import load
+import safetensors
+
+model = torch.load('model.pth') # Noncompliant {{Replace this call with a safe alternative}}
+       #^^^^^^^^^^
+
+some_path = ...
+model = load(some_path) # Noncompliant
+
+torch.load(model2, 'model.pth', weights_only=False) #Noncompliant
+
+model2 = ...
+safetensors.torch.load_model(model2, 'model.pth')
+
+torch.load(model2, 'model.pth', weights_only=True)
+torch.load(model2, 'model.pth', weights_only=some_value)
```

Comment:
Here what if `some_value` is set to `False`. You could maybe add a test for that. I think it would be a good idea to raise in such a case. I would suggest to look into `ReachingDefinitionAnalysis` in order to figure out how we can check the assigned value of `some_value`. I think it make sense to go that road but I would not go deeper for example if a function is called for the `weights_only` parameter. I would still add it as a test to better show the intent to the next person who will read this rule. ```torch.load(..., weights_only=foo()) # Compliant```.